src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.css

```diff
@@ -67,6 +67,10 @@
   color: var(--side-nav-hover-text-color) !important;
 }
 
+:deep(.shaka-current-selection-span) {
```

Comment:
@absidue This also affects the color of the stats button text, which can render it nearly invisible depending on your theme.
<img width="521" alt="Screen Shot 2024-12-09 at 11 28 20 am" src="https://github.com/user-attachments/assets/98745218-c11e-4c8c-b68c-f026d738db84">

How about:

```css
:deep(.shaka-overflow-menu-only .shaka-current-selection-span) {
```

Reply:
@OothecaPickle could you open up an issue for this so this doesnt get lost

Reply:
> @OothecaPickle could you open up an issue for this so this doesnt get lost

#6406